modules/aws-ext/README.txt

```diff
@@ -6,7 +6,7 @@ Apache Ignite AWS module provides S3-based implementations of checkpoint SPI and
 Depending on how you use Ignite, you can an extension using one of the following methods:
 
 - If you use the binary distribution, move the lib/optional/{module-dir} to the lib directory before starting the node.
-- Add libraries from lib/optional/{module-dir} to the classpath of your application.
+- Add libraries from lib/{module-dir} to the classpath of your application.
```

Comment:
There is no `lib/{module-dir}` directory in the assembly, all files are inside `lib` directory.
Also, `If you use the binary distribution, move the lib/optional/{module-dir} to the lib directory before starting the node.` should be fixed too.